pkg/pillar/scripts/device-steps.sh

```diff
@@ -164,6 +167,25 @@ if [ ! -d $PERSISTDIR/log ]; then
     mkdir $PERSISTDIR/log
 fi
 
+
+LAST_EVE_VERSION=""
+if [ -f $LAST_IMAGE_VERSION_FILE ];
+then
+     LAST_EVE_VERSION=$(jq '.Version' < $LAST_IMAGE_VERSION_FILE| tr -d '"')
+fi
```

Comment:
This gets me back to the naming and the content of the file. I would appreciate if the name were to reflect that it simply tracks the last version of (either EVE or /persist schema) and then that's what the content is. So there's no need for JSON format -- it simply is a string.

Reply:
Thought Having Time as well as Version string is useful.. And may be we can add more info going forward.. This is currently generated by baseos mgr because that is the service that confirms the upgrade is done..